CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -850,9 +850,6 @@ CF_INLINE const char *CFPathRelativeToAppleFrameworksRoot(const char *path, Bool
 #if (DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_EMBEDDED_MINI || DEPLOYMENT_TARGET_LINUX)
 #if __HAS_DISPATCH__
 
-#include <dispatch/dispatch.h>
-#include <dispatch/private.h>
```

Comment:
We don't have a dispatch/private.h on Linux so it needs to be removed (at least for now). There's also no direct need to reference dispatch/dispatch.h from CFInternal.h, but I'll, update to retain the reference.